src/wast-parser.cc

```diff
@@ -1495,6 +1496,17 @@ Result WastParser::ParsePlainInstr(std::unique_ptr<Expr>* out_expr) {
       break;
     }
 
+    case TokenType::Simd1LaneImm: {
+      Token token = Consume();
+      ErrorUnlessOpcodeEnabled(token);
+      Const const_;
+      CHECK_RESULT((ParseSimdConstWithCheck<uint32_t>(token.opcode(), Type::I32,
```

Comment:
Why parse as SimdConst? This should just use `ParseNat` to parse a natural number.

Reply:
Still Consider the future v8x16.shuffle implementation.

ParseNat can only parse 1 natural number. But ParseSimdConst can parse several natural numbers according to the input byte numbers.

Reply:
OK, this seems a bit overly-generic to me, but I'm OK with this direction for now. We can always refactor later if necessary.

Reply:
Done.
use ParsrNat for simd extract/replace lane as v8x16.shuffle will be another Expr type.